assembl/static2/js/app/components/common/tags/tags.jsx

```diff
@@ -0,0 +1,161 @@
+// @flow
+
+import React, { Component } from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { WithContext as ReactTags } from 'react-tag-input';
+/* eslint-enable */
+import { compose, graphql } from 'react-apollo';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+// Component imports
+import AddkeywordIcon from '../icons/addkeywordIcon/addkeywordIcon';
+import CrossIcon from '../icons/crossIcon/crossIcon';
+import manageErrorAndLoading from '../manageErrorAndLoading';
+// Helper imports
+import { displayAlert } from '../../../utils/utilityManager';
+import { formatedTagList } from '../../../utils/globalFunctions';
+// GraphQL imports
+import addTagMutation from '../../../graphql/mutations/addTag.graphql';
+import removeTagMutation from '../../../graphql/mutations/removeTag.graphql';
+
+export type TagProps = {
+  /** Tag ID */
+  id: string,
+  /** Tag value */
+  text: string
+};
+
+export type Props = {
+  /** Flag that checks whether the tag is already added and displayed */
+  alreadyAdded?: string,
```

Comment:
boolean ?

Reply:
Ok after reading the rest of the component I got what it's about. I'd call this alreadyAddedWarningMessage or something, the current name is a bit confusing